src/edu.umn.cs.melt.copper.compiletime/src/main/java/edu/umn/cs/melt/copper/main/ParserCompilerParameters.java

```diff
@@ -192,6 +202,8 @@ public ParserCompilerParameters()
 		dumpFile = null;
 		dumpOutputType = null;
 		avoidRecompile = false;
+		colorCounterexample = true;
```

Comment:
Would it be easy to make the default be to check whether stdout supports ANSI colors?

Reply:
The only way I know of to check terminal capabilities like this is OS-specific (reading the `TERM` environment variable). What could be done is to default to "no color" when the output is to a log file.

Also, I see that the Javadoc in a few places refers to the ANSI codes as "ascii color codes". This should be corrected as it goes into Copper's public documentation.

Reply:
Fixed the color code names

Reply:
Changed, I had to add another flag to enable color to accommodate this, which I felt was better than not having an override to this behavior, or having a flag that takes a boolean as argument and needing to turn a string in to a boolean.

Reply:
We really should rework this so that the "default" color setting is the same whether invoking Copper through the CLI or the API, butwe do not need to hold up the merge on this account.